build.gradle

```diff
@@ -112,18 +114,25 @@ allprojects {
 		}
 	}
 
-	task runTestmodClient(type: RunClientTask) {
-		classpath sourceSets.testmod.runtimeClasspath
-	}
-
-	task runTestmodServer(type: RunServerTask) {
-		classpath sourceSets.testmod.runtimeClasspath
+	loom {
+		runs {
+			testmodClient {
+				client()
+				ideConfigGenerated false
```

Comment:
Why not generate an ide run config?

Reply:
Becuase you would have over 100 of them, should just do that for the root project thinking about it.

Reply:
Done.